### PR TITLE
5.0.2 eye form

### PR DESCRIPTION
**Pull request by ophthal**
Ok...

### CODE REVIEW

File: interface/forms/eye_mag/php/taskman_functions.php [hunk 1]
```diff
@@ -4,11 +4,25 @@
...
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @package OpenEMR
+ * @author Ray Magauran <magauran@MedFetch.com>
+ * @link http://www.open-emr.org
  */
```
Comment (ophthal) [inline interface/forms/eye_mag/php/taskman_functions.php]: This can be reverted.

File: interface/forms/eye_mag/SpectacleRx.php [hunk 2]
```diff
@@ -1296,7 +1295,7 @@ class="prism"
 
 </body>
 <script>
-    $(function () {
+    $(document).ready(function () {
```
Comment (bradymiller) [inline interface/forms/eye_mag/SpectacleRx.php]: revert above line

File: interface/forms/eye_mag/a_issue.php [hunk 3]
```diff
@@ -164,7 +182,7 @@
 
             if ($local =="1") { // leave FH/SocHx/ROS for later - done below separately
                 while ($res = sqlFetchArray($qry)) { //Should we take the top 10 and display alphabetically?
-                    echo " aopts['" .attr($key). "'][aopts['" .attr($key). "'].length] = new Option('".attr(trim($res['option_id']))."', '".attr(xl_list_label(trim($res['title'])))."', false, false);\n";
+                    echo " aopts['" .attr($key). "'][aopts['" .attr($key). "'].length] = new Option('".attr(xl_list_label(trim($res['title'])))."', '".attr(trim($res['option_id']))."', false, false);\n";
```
Comment (bradymiller) [inline interface/forms/eye_mag/a_issue.php]: For above line, can do the following to make it more solid since this is within javascript context:
```
echo " aopts['" .attr($key). "'][aopts['" .attr($key). "'].length] = new Option(".js_escape(xl_list_label(trim($res['title']))).", ".js_escape(trim($res['option_id'])).", false, false);\n";
```
(note that I used the js_escape and removed the single quotes surrounding them)

File: interface/forms/eye_mag/php/Anything_simple.php [hunk 4]
```diff
@@ -1,590 +0,0 @@
-<?php
-/**
```
Comment (bradymiller) [inline interface/forms/eye_mag/php/Anything_simple.php]: fare thee well Anything_simple.php
:)

File: interface/forms/eye_mag/php/eye_mag_functions.php [hunk 5]
```diff
@@ -2104,12 +2107,36 @@ function display_PMSFH($rows, $view = "pending", $min_height = "min-height:344px
...
+                if ( ($key == "Medication") && ( !empty($item['row_subtype'])) ) {
+                    $subtype_Meds[$item['row_subtype']]['name'] = $item['row_subtype'];
+                    $subtype_Meds[$item['row_subtype']]['header'] = '
+                        <table class="PMSFH_header">
+                            <tr>
+                                <td width="90%">
+                                    <span class="left" style="font-weight:800;font-size:0.9em;">'.xlt(ucwords($item['row_subtype'])).' Meds</span>
+                                </td>
+                                <td>
+                                    <span class="right btn-sm" href="#PMH_anchor" onclick="alter_issue2(\'0\',\''.attr($key).'\',\'0\');" style="text-align:right;font-size:8px;">'. xlt("New") .'</span>
```
Comment (bradymiller) [inline interface/forms/eye_mag/php/eye_mag_functions.php]: We have a crazy escaping function for when js is embedded in html like the onclick. Can do the following here:
```
<span class="right btn-sm" href="#PMH_anchor" onclick="alter_issue2(\'0\','.attr_js($key).',\'0\');" style="text-align:right;font-size:8px;">'. xlt("New") .'</span>
```
(note used attr_js function and removed the surrounding quotes)

File: interface/forms/eye_mag/php/eye_mag_functions.php [hunk 6]
```diff
@@ -2104,12 +2107,36 @@ function display_PMSFH($rows, $view = "pending", $min_height = "min-height:344px
...
+                                </td>
+                            </tr>
+                        </table>
+                        ';
+                    $subtype_Meds[$item['row_subtype']]['table'] .= "<span $red name='QP_PMH_".$item['rowid']."' href='#PMH_anchor' id='QP_PMH_".$item['rowid']."'
```
Comment (bradymiller) [inline interface/forms/eye_mag/php/eye_mag_functions.php]: attr
```
$subtype_Meds[$item['row_subtype']]['table'] .= "<span $red name='QP_PMH_".attr($item['rowid'])."' href='#PMH_anchor' id='QP_PMH_".attr($item['rowid'])."'
```

File: interface/forms/eye_mag/php/eye_mag_functions.php [hunk 7]
```diff
@@ -2104,12 +2107,36 @@ function display_PMSFH($rows, $view = "pending", $min_height = "min-height:344px
...
+                onclick=\"alter_issue2('".attr($item['rowid'])."','".attr($key)."','".attr($index)."');\">".text($item['title']).$reaction."</span><br />";
```
Comment (bradymiller) [inline interface/forms/eye_mag/php/eye_mag_functions.php]: more for the crazy attr_js function (and note I removed the surrounding single quotes):
```
onclick=\"alter_issue2(".attr_js($item['rowid']).",".attr_js($key).",".attr_js($index).");\">".text($item['title']).$reaction."</span><br />";
```

Comment (bradymiller) [reply interface/forms/eye_mag/php/eye_mag_functions.php]: Also, if $reaction doesn't contain html in in, then it should be escaped by including it in the text call:
```
text($item['title'].$reaction)
```

File: interface/forms/eye_mag/php/eye_mag_functions.php [hunk 8]
```diff
@@ -2369,7 +2404,30 @@ function show_PMSFH_panel($PMSFH, $columns = '1')
...
+    <?php
+    if ($PMSFH[0]['Eye Meds']) {
+         $i=0;
+        foreach ($PMSFH[0]['Eye Meds'] as $item) {
+             if ( ($item['status'] == "Inactive") || ($item['row_subtype'] != "eye") ){
+                $i++;
+                continue;
+            }
+            echo "<span name='QP_PMH_".attr($item['rowid'])."' href='#PMH_anchor' id='QP_PMH_".attr($item['rowid'])."'
+            onclick=\"alter_issue2('".attr(addslashes($item['rowid']))."','Medication','$i');\">".text($item['title'])."</span><br />";
```
Comment (bradymiller) [inline interface/forms/eye_mag/php/eye_mag_functions.php]: more for the crazy attr_js function (and note I removed the surrounding single quotes)(note it is much better than addslashes):
```
onclick=\"alter_issue2(".attr_js($item['rowid']).",'Medication','$i');\">".text($item['title'])."</span><br />";
```

File: interface/forms/eye_mag/php/taskman_functions.php (hunk 1 above)
Comment (bradymiller) [reply interface/forms/eye_mag/php/taskman_functions.php]: yes, revert above

File: interface/forms/eye_mag/view.php [hunk 9]
```diff
@@ -174,6 +173,35 @@
...
+              <span>Ant</span>
+          </li>
+          <li id="tabs-left-RETINA" class="btn-primary">
+              <span>Retina</span>
+          </li>
+          <li id="tabs-left-NEURO" class="btn-primary">
+              <span>Neuro</span>
+          </li>
+          <li id="tabs-left-IMPPLAN" class="btn-primary">
+              <span>Imp</span>
```
Comment (bradymiller) [inline interface/forms/eye_mag/view.php]: Need to translate these labels

File: interface/forms/eye_mag/view.php [hunk 10]
```diff
@@ -186,10 +214,23 @@
...
         ?>
-
-
-        <!-- start form -->
+          <script>
+              var Code_new_est ='<?php
+                 if  ($output_priors =='') {
+                     echo "New";
+                 } else {
+                     echo "Est";
```
Comment (bradymiller) [inline interface/forms/eye_mag/view.php]: translate labels

File: interface/themes/style_manila.css [hunk 11]
```diff
@@ -0,0 +1,1897 @@
+/*! This style sheet was autogenerated using gulp + scss
```
Comment (bradymiller) [inline interface/themes/style_manila.css]: Note this style_manila.css file no longer exists. Are you needing to modify the manila theme for your work?

**Comment by bradymiller** [comment]
hi @ophthal ,
Just completed the code review. Let me know when ready for another review (at that time should be ready for codebase). Also want to ensure there are no database changes. thanks, -brady

File: interface/forms/eye_mag/view.php (hunk 10 above)
Comment (ophthal) [reply interface/forms/eye_mag/view.php]: submitting this as 
echo xla("New");

This will end up as javascript.  Does it need one of these new js_escape functions too?

File: interface/themes/style_manila.css (hunk 11 above)
Comment (ophthal) [reply interface/themes/style_manila.css]: But I thought the sass file was built off of this then run through gulp.
When in a docker I change this style_manilla.css file and reload it with npn run dev.
So is it really gone?  It was the only style sheet with "non-flat" styling IMHO.

**Comment by ophthal** [comment]
All set - and yes if the manila style exists somewhere, I'd like these changes incorporated in it if possible.  Shouldn't break anything.  Hopefully some of the features in manila that make it different from the flat themes make it into the codebase.

File: interface/forms/eye_mag/view.php (hunk 10 above)
Comment (bradymiller) [reply interface/forms/eye_mag/view.php]: No, wouldn't worry about using it here and just use xls(). Can only use the js escaping functions at the endpoint since it places quotes around it in js context.

File: interface/themes/style_manila.css (hunk 11 above)
Comment (bradymiller) [reply interface/themes/style_manila.css]: manila style is still there, but in sass form. the .css file gets built in the build process.

Comment (bradymiller) [reply interface/themes/style_manila.css]: hi @zbig01 ,
Was hoping you could help provide guidance for @ophthal on best way to get his styling changes into the scss form. -brady

**Comment by zbig01** [comment]
@ophthal  @bradymiller

FYI there are three basic stylesheets - light, manilla and color/flat. If you choose to add styles to all three stylesheets they would need to be added in` interface/themes/oe-common` directory, create a scss file called say `eye-module.scss`, add all the rule-sets you want. Add a line in `interface/themes/oe-common/all-common-import.scss file - @import eye-module`.
 
Execute `npm-run-build` to add these rule-sets to all the stylesheets.

You should consider adding rule-sets to a specific stylesheet only if there is a major difference in the way it is being rendered on that stylesheets.

HOT TIP: In case you are using float rules in your styling you can avoid rtl issues by using the 2 css classes - `oe-pull-away` and `oe-pull-toward` . In ltr languages `oe-pull-away {float: right}, oe-pull-toward {float:left}` and vice-versa in rtl languages.

As there already is a separate css file for the eye module would it not make sense to add these rule-sets to that css stylesheet instead?

**Comment by bradymiller** [comment]
btw @ophthal , don't worry about the merge conflicts (we just brought in commits into the codebase that did a lot of minor changes int he entire codebase). i'll fix the merge conflicts when this pr is ready for the codebase.

File: interface/forms/eye_mag/SpectacleRx.php [hunk 12]
```diff
@@ -61,656 +60,645 @@
...
-            text-decoration: unset;
-        }
-        table {
-            font-size: 1.0em;
-            padding: 12px;
-            color: black;
-            vertical-align: text-top;
-        }
+        <script type="text/javascript" src="<?php echo $webroot ?>/interface/main/tabs/js/include_opener.js"></script>
+        <?php Header::setupHeader([ 'jquery-ui', 'jquery-ui-redmond','pure', 'bootstrap' ,'jscolor' ]); ?>
```
Comment (bradymiller) [inline interface/forms/eye_mag/SpectacleRx.php]: bootstrap can be remove from above since it is always included
can also add 'opener' in line above and then remove the line at 291 (include_opener.js include)
you should also be able to remove line 294 (css_header include) since this is always included in above line

File: interface/forms/eye_mag/SpectacleRx.php [hunk 13]
```diff
@@ -61,656 +60,645 @@
...
-        $fields['REFDATE'] = $data['date'];
-        $insert_this_id = formSubmit($table_name, $fields, $form_id, $_SESSION['userauthorized']);
     }
-}
-
-if ($_REQUEST['dispensed']) {
-    $query = "SELECT * from form_eye_mag_dispense where pid =? ORDER BY date DESC";
-    $dispensed = sqlStatement($query, array($_REQUEST['pid']));
-    ?><html>
+   // $GLOBALS['assets_static_relative']= "/openemr/public/assets";
```
Comment (bradymiller) [inline interface/forms/eye_mag/SpectacleRx.php]: rec removing above comment

File: interface/forms/eye_mag/SpectacleRx.php [hunk 14]
```diff
@@ -61,656 +60,645 @@
...
+        <?php
+        exit;
+    }
+    
+    ob_start();
+?>
+<html>
+<head>
+    <script type="text/javascript" src="<?php echo $webroot ?>/interface/main/tabs/js/include_opener.js"></script>
+    <?php Header::setupHeader([ 'jquery-ui', 'jquery-ui-redmond','pure', 'bootstrap' ]); ?>
```
Comment (bradymiller) [inline interface/forms/eye_mag/SpectacleRx.php]: same advice here as above post

File: interface/forms/eye_mag/SpectacleRx.php [hunk 15]
```diff
@@ -1296,7 +1284,7 @@ class="prism"
 
 </body>
 <script>
-    $(function () {
+    $(document).ready(function () {
```
Comment (bradymiller) [inline interface/forms/eye_mag/SpectacleRx.php]: revert this change

File: interface/forms/eye_mag/php/eye_mag_functions.php (hunk 5 above)
Comment (bradymiller) [reply interface/forms/eye_mag/php/eye_mag_functions.php]: need to remove the surrounding quotes on this attr_js call:
```
<span class="right btn-sm" href="#PMH_anchor" onclick="alter_issue2(\'0\','.attr_js($key).',\'0\');" style="text-align:right;font-size:8px;">'. xlt("New") .'</span>
```

File: interface/forms/eye_mag/php/eye_mag_functions.php (hunk 8 above)
Comment (bradymiller) [reply interface/forms/eye_mag/php/eye_mag_functions.php]: this change has not yet been addressed

File: interface/forms/eye_mag/php/eye_mag_functions.php [hunk 16]
```diff
@@ -2369,7 +2411,30 @@ function show_PMSFH_panel($PMSFH, $columns = '1')
         onclick="alter_issue2('0','POS','');" class="disabled_button"><?php echo xlt("None"); ?><br /></span>
         <?php
     }
-
+    
+//<!-- Eye Meds -->
+    echo "<br /><span class='panel_title' title='".xlt("Eye Meds")."'>".xlt("Eye Meds").":</span>";
```
Comment (bradymiller) [inline interface/forms/eye_mag/php/eye_mag_functions.php]: first xlt in this line should be xla

File: interface/forms/eye_mag/php/eye_mag_functions.php [hunk 17]
```diff
@@ -4577,19 +4733,20 @@ function display_GlaucomaFlowSheet($pid, $bywhat = 'byday')
...
-    $count_OCT = empty($documents['docs_in_name']['OCT']) ? 0 : count($documents['docs_in_name']['OCT']);
+    $count_OCT = count((array)$documents['docs_in_name']['OCT']);
     if ($count_OCT > 0) {
         foreach ($documents['docs_in_name']['OCT'] as $OCT) {
             $OCT_date[] = $OCT['docdate'];
         }
     }
 
-    $count_VF = empty($documents['docs_in_name']['VF']) ? 0 : count($documents['docs_in_name']['VF']);
+    $count_VF = count((array)$documents['docs_in_name']['VF']);
```
Comment (bradymiller) [inline interface/forms/eye_mag/php/eye_mag_functions.php]: revert this line since it was addressing a php warning (the uncountable warning)

File: interface/forms/eye_mag/php/eye_mag_functions.php [hunk 18]
```diff
@@ -4577,19 +4733,20 @@ function display_GlaucomaFlowSheet($pid, $bywhat = 'byday')
...
     global $OSIOPTARGET;
     global $dated;
+    global $visit_date;
 
     if (!$documents) {
         list($documents) = document_engine($pid);
     }
 
-    $count_OCT = empty($documents['docs_in_name']['OCT']) ? 0 : count($documents['docs_in_name']['OCT']);
+    $count_OCT = count((array)$documents['docs_in_name']['OCT']);
```
[the remark this answers is not in the document]
Comment (bradymiller) [reply interface/forms/eye_mag/php/eye_mag_functions.php]: unless your change also doesn't cause the warning

File: interface/forms/eye_mag/php/eye_mag_functions.php (hunk 17 above)
Comment (bradymiller) [reply interface/forms/eye_mag/php/eye_mag_functions.php]: unless your change also doesn't cause the warning, then keep your change

File: interface/forms/eye_mag/php/eye_mag_functions.php [hunk 19]
```diff
@@ -4741,13 +4898,11 @@ function display_GlaucomaFlowSheet($pid, $bywhat = 'byday')
...
 
-    for ($a=0; $a < count($date_OU); $a++) {
-        if (!empty($GONIO_date)) {
-            foreach ($GONIO_date as $GONIO) {
-                if ($date_OU[$a] == $GONIO) {
-                    $GONIO_values[$a] = "1";
-                    break;
-                }
+    for ($a=0; $a < count((array)$date_OU); $a++) {
+        foreach ((array)$GONIO_date as $GONIO) {
```
Comment (bradymiller) [inline interface/forms/eye_mag/php/eye_mag_functions.php]: ensure this change does not bring back the uncountable php warning

File: interface/forms/eye_mag/php/taskman_functions.php (hunk 1 above)
Comment (bradymiller) [reply interface/forms/eye_mag/php/taskman_functions.php]: still need to revert above

File: interface/forms/eye_mag/report.php [hunk 20]
```diff
@@ -607,11 +606,16 @@ function narrative($pid, $encounter, $cols, $form_id, $choice = 'full')
...
-              <td><?php echo text($CONTRASTOSVA); ?></td>
-            </tr>
+                <tr>
+                    <td><?php echo xlt('Contrast{{Constrast Visual Acuity}}'); ?></td>
+                    <td><?php echo text($CONTRASTODVA); ?></td>
+                    <td><?php echo text($CONTRASTOSVA); ?></td>
+                </tr>
+            <?php } if (!empty($BINOCVA)) { ?>
+                <tr>
+                  <td><?php echo xlt('VABiNoc{{|Binocular Visual Acuity}}'); ?></td>
```
Comment (bradymiller) [inline interface/forms/eye_mag/report.php]: Just checking if you meant to have that `|` character in there to the left of the Binocular.

File: interface/forms/eye_mag/save.php [hunk 21]
```diff
@@ -215,6 +215,20 @@
...
+// mdsupport - user_settings prefix
+    $uspfx = "EyeFormSettings_";
+    $setting_tabs_left = prevSetting($uspfx, 'setting_tabs_left', 'setting_tabs_left', '0');
+    $setting_HPI = prevSetting($uspfx, 'setting_HPI', 'setting_HPI', '1');
+    $setting_PMH = prevSetting($uspfx, 'setting_PMH', 'setting_PMH', '1');
+    $setting_ANTSEG = prevSetting($uspfx, 'setting_ANTSEG', 'setting_ANTSEG', '1');
+    $setting_POSTSEG = prevSetting($uspfx, 'setting_POSTSEG', 'setting_POSTSEG', '1');
+    $setting_EXT = prevSetting($uspfx, 'setting_EXT', 'setting_EXT', '1');
+    $setting_NEURO = prevSetting($uspfx, 'setting_NEURO', 'setting_NEURO', '1');
+    $setting_IMPPLAN = prevSetting($uspfx, 'setting_IMPPLAN', 'setting_IMPPLAN', '1');
```
Comment (bradymiller) [inline interface/forms/eye_mag/save.php]: always cool to see use of sticky settings

File: interface/forms/eye_mag/view.php (hunk 9 above)
Comment (bradymiller) [reply interface/forms/eye_mag/view.php]: Still need to translate these labels

File: interface/forms/eye_mag/view.php (hunk 10 above)
Comment (bradymiller) [reply interface/forms/eye_mag/view.php]: Still need to translate these labels

File: interface/forms/eye_mag/view.php [hunk 22]
```diff
@@ -246,10 +299,16 @@
...
-
-              <!-- start first div -->
+                <input type='hidden' name='setting_tabs_left' id='setting_tabs_left' value='<?php echo $setting_tabs_left; ?>'>
+                <input type='hidden' name='setting_HPI' id='setting_HPI' value='<?php echo $setting_HPI; ?>'>
+                <input type='hidden' name='setting_PMH' id='setting_PMH' value='<?php echo $setting_PMH; ?>'>
+                <input type='hidden' name='setting_EXT' id='setting_EXT' value='<?php echo $setting_EXT; ?>'>
+                <input type='hidden' name='setting_ANTSEG' id='setting_ANTSEG' value='<?php echo $setting_ANTSEG; ?>'>
+                <input type='hidden' name='setting_POSTSEG' id='setting_POSTSEG' value='<?php echo $setting_POSTSEG; ?>'>
+                <input type='hidden' name='setting_NEURO' id='setting_NEURO' value='<?php echo $setting_NEURO; ?>'>
+                <input type='hidden' name='setting_IMPPLAN' id='setting_IMPPLAN' value='<?php echo $setting_IMPPLAN; ?>'>
```
Comment (bradymiller) [inline interface/forms/eye_mag/view.php]: need to wrap all the above (8 lines) echo variables with attr()

File: sql/5_0_1-to-5_0_2_upgrade.sql [hunk 23]
```diff
@@ -230,6 +230,7 @@ CREATE TABLE `form_eye_acuity` (
   `LIOSVA`        varchar(25) NOT NULL,
   `WODVANEAR`     varchar(25)  DEFAULT NULL,
   `OSVANEARCC`    varchar(25)  DEFAULT NULL,
+  `BINOCVA`       varchar(25)  DEFAULT NULL,
```
Comment (bradymiller) [inline sql/5_0_1-to-5_0_2_upgrade.sql]: Would add this at the bottom of the upgrade script with it's own If block (otherwise folks that are running from most recent codebase won't get this change)

File: sql/database.sql [hunk 24]
```diff
@@ -8817,6 +8821,7 @@ INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`
 ('Eye_Defaults_for_GENERAL', 'OSIRIS', 'round', 240, 0, 0, '', 'ANTSEG', '', 0, 0, 1, ''),
 ('Eye_Defaults_for_GENERAL', 'OSLENS', 'clear', 220, 0, 0, '', 'ANTSEG', '', 0, 0, 1, ''),
 ('Eye_Defaults_for_GENERAL', 'OSMACULA', 'flat', 480, 0, 0, '', 'RETINA', '', 0, 0, 1, ''),
+('Eye_Defaults_for_GENERAL', 'OSPERIPH', 'clear', 515, 0, 0, '', 'RETINA', '', 0, 0, 1, ''),
```
Comment (bradymiller) [inline sql/database.sql]: above new entries need to be added in sql upgrade script. Let me know if you need me to show and example of how to add a new list_options item in the sql upgrade script.

**Comment by bradymiller** [comment]
Code review complete. I am planning to look at the interface/themes/style_manila.css in more detail (will compare it to the current style_manila.css that is built to see how extensive the changes are there.

**Comment by bradymiller** [comment]
also, to make it easier to see my comments and see non-whitespace changes, use this link:
https://github.com/openemr/openemr/pull/2525/files?w=1

File: interface/forms/eye_mag/report.php (hunk 20 above)
Comment (ophthal) [reply interface/forms/eye_mag/report.php]: No.  Will fix this

File: interface/forms/eye_mag/SpectacleRx.php (hunk 12 above)
Comment (ophthal) [reply interface/forms/eye_mag/SpectacleRx.php]: done

File: interface/forms/eye_mag/SpectacleRx.php (hunk 15 above)
Comment (ophthal) [reply interface/forms/eye_mag/SpectacleRx.php]: done

File: interface/forms/eye_mag/SpectacleRx.php (hunk 13 above)
Comment (bradymiller) [reply interface/forms/eye_mag/SpectacleRx.php]: above comment not yet addressed

File: interface/forms/eye_mag/SpectacleRx.php (hunk 14 above)
Comment (bradymiller) [reply interface/forms/eye_mag/SpectacleRx.php]: above comment not yet addressed

File: interface/forms/eye_mag/view.php (hunk 9 above)
Comment (bradymiller) [reply interface/forms/eye_mag/view.php]: above comment not yet addressed

File: interface/forms/eye_mag/view.php (hunk 10 above)
Comment (bradymiller) [reply interface/forms/eye_mag/view.php]: above comment not yet addressed

File: interface/forms/eye_mag/view.php (hunk 22 above)
Comment (bradymiller) [reply interface/forms/eye_mag/view.php]: above comment not yet addressed

File: sql/5_0_1-to-5_0_2_upgrade.sql (hunk 23 above)
Comment (bradymiller) [reply sql/5_0_1-to-5_0_2_upgrade.sql]: above comment not yet addressed

File: sql/database.sql (hunk 24 above)
Comment (bradymiller) [reply sql/database.sql]: above comment not yet addressed

**Comment by bradymiller** [comment]
hi @ophthal , pointed out several issues that have not yet been addressed.

**Comment by bradymiller** [comment]
rebased and continued on this PR:
https://github.com/openemr/openemr/pull/2576
